### DISCUSSION TITLE
Welcome to Serverpod Discussions!

### DISCUSSION THREAD

**Discussion by vlidholt** [serverpod/serverpod#22] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by anandsubbu007** [comment]
hi there, i like to contribute to this plugin. but i cant able to set environment in my device. when i run pubspec generator it doesn't modify files in templates/pubspec folder. 
may i know the how to setup this dev branch i my device.

> serverpod generate-pubspecs --version 0.9.7 --mode production   

i like to do that what i can do...

```
Because every version of serverpod_auth_server from path depends on serverpod from path and subbu_shop_server depends on serverpod from hosted, serverpod_auth_server from path is forbidden.
So, because subbu_shop_server depends on serverpod_auth_server from path, version solving failed.
```

**Reply by vlidholt** [reply]
The development scripts are only tested on MacOS. Typically, you will want to run the generation of pubspecs through the script in the `util` folder

    $ util/update_pubspecs

**Comment by ordru** [comment]
Hello everyone,

**About myself**

My name is ordru. My experience with dart and fluter stems from developing different applications in the past. I am interested in backend technologies, especially scalable real-time servers, etc. I have been using Firebase, AppWrite, Supabase and others technologies in the past. I think that having a unified solution for the frontend and backend and tools, all written in the same language and having a framework that is highly modular adaptable and scalable, makes Serverpod for me what it is and stand out from other solutions. It's this why I am drawn to the project. 

**About how I would like to contribute**

Unfortunately time is a really big constrain for me, but as time allows me, I would like to contribute to the project by participating to ongoing discussions on various topics related to communication & development, proposing features, focusing on updating documentation, at times reviewing changes, and hopefully contribute by implementing various features.
